run.py

```diff
@@ -883,6 +888,18 @@ def main():
     print("FAILED:  %d" % (failed))
     print("SKIPPED: %d" % (skipped))
 
+    if args.debugfailed:
+        if len(failedLogs) > 0:
+            print("")
+            print("Failed tests debug output:")
+        for dirpath in failedLogs:
+            print("- Test %s:" % os.path.basename(dirpath))
+            for r, d, f in os.walk(dirpath+"/output"):
+                for fname in f:
+                    print("  - %s" % fname)
+                    fcontents = open(dirpath+"/output/"+fname, "r")
+                    print(fcontents.read())
+
```

Comment:
So, just to be sure, we want to dump the information about why a test failed after running all the tests and not right when they fail?

Reply:
Hmm... we could do it right away
This is a design choice about how a human would better read it